### PR TITLE
Translation of the application into ru lang

### PR DESCRIPTION
**Pull request by Marshal-Odessa**
photo_2022-09-25_07-25-34 (3)
photo_2022-09-25_07-25-47
photo_2022-09-25_07-25-34
photo_2022-09-25_07-25-34 (
![photo_2022-09-25_07-25-47
2)](https://user-images.githubusercontent.com/48080818/192128387-1819b34b-2cde-481a-a361-d0bc02f72351.jpg)

### CODE REVIEW

**Comment by z-huang** [comment]
Build failed. Please fix it.

File: app/src/main/res/values-ru/strings.xml [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+    <string name="pref_backup_restore_title">Бэкап и восстановление</string>
+    <string name="pref_backup_title">Резервное копирование</string>
+    <string name="pref_restore_title">Восстановить</string>
+    <string name="pref_about_title">О приложении</string>
+    <string name="pref_app_version_title">Версия</string>
+    <string name="pref_github_title" translatable="false">GitHub</string>
+    <string name="pref_github_summary" translatable="false">z-huang/music</string>
+
+    <!-- Colors -->
+    <string name="color_sakura">Sakura</string>
```
Comment (chirukodati) [inline app/src/main/res/values-ru/strings.xml:55]: This also needs to be translated.

**Comment by z-huang** [comment]
You should pass the build before I merge the PR.

**Comment by z-huang** [comment]
Stale for too long. Please open another PR when you're ready.

**Pull request closed without merging**